endgame_postprocessing/post_processing/aggregation.py

```diff
@@ -266,28 +293,19 @@ def country_lvl_aggregate(
             "Threshold summary measures are required to be input."
         )
 
-    summarize_threshold = _threshold_summary_helper(
-        processed_iu_lvl_data,
-        threshold_summary_measure_names,
-        threshold_groupby_cols,
-        partial(_calc_count_of_non_na_or_negative, denominator_val=denominator_to_use),
-        "pct_of_",
-        threshold_cols_rename,
-    )
-
-    summarize_threshold_counts = _threshold_summary_helper(
-        processed_iu_lvl_data,
-        threshold_summary_measure_names,
-        threshold_groupby_cols,
-        _calc_count_of_non_na_or_negative,
-        "count_of_",
-        threshold_cols_rename,
-    )
+    yearly_pct_of_runs_dfs = []
+    for pct in pct_runs_under_threshold:
+        yearly_pct_of_runs_dfs.append(_yearly_pct_of_runs_threshold_summary_helper(
+            processed_iu_lvl_data,
+            threshold_groupby_cols,
+            denominator_to_use,
+            pct
+        ))
 
     summarize_threshold_year = _threshold_summary_helper(
         processed_iu_lvl_data,
         threshold_summary_measure_names,
-        threshold_groupby_cols,
+        [col for col in threshold_groupby_cols if col != canoncical_columns.YEAR_ID],
```

Comment:
💡 Probably easier to leave this as just thrshold_groupy_col as is, and `+ [canoncical_columns.YEAR_ID]` in `_yearly_pct_of_runs_threshold_summary_helper` - though IMO I would be tempted to inline them where we use them as I think this makes it harder to see what is going on.